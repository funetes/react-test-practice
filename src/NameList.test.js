import { render } from '@testing-library/react';
import React from 'react';
import randerer from 'react-test-renderer';
import NameList from './NameList';

describe('NameListComponet', () => {
  let component = null;
  it('renders correctly', () => {
    component = randerer.create(<NameList names={['벨로퍼트', '김민준']} />);
  });
  it('match sanpshot', () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
