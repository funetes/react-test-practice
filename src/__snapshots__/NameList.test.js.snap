// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NameListComponet match sanpshot 1`] = `
<ul>
  <li>
    벨로퍼트
  </li>
  <li>
    김민준
  </li>
</ul>
`;
